ffi/dart/test/icu_test.dart

```diff
@@ -9,31 +9,6 @@ void main() {
     expect(x.toString(), '1.4940300');
   });
 
-  test('LocaleFallbacker', () {
```

Comment:
don't remove tests, these are failing because the binary is built without the relevant features. the icu4x publish should build with default_components and *maybe* experimental as well